### PR TITLE
Nationalcloud

### PR DESCRIPTION
**Pull request by hforeste**

### CODE REVIEW

File: src/Diagnostics.RuntimeHost/Services/GithubClient/AzureStorageSourceCodeClient.cs [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+            }
+
+            foreach (var commit in commits)
+            {
+                if (!Equals(commit.EncodingType, EncodingType.Base64))
+                {
+                    commit.Content = InternalAPIHelper.Base64Encode(commit.Content);
+                }
+                await _storageService.LoadBlobToContainer(commit.FilePath, commit.Content).ConfigureAwait(false);
+            }
```
Comment (rekhaswaminathan) [inline src/Diagnostics.RuntimeHost/Services/GithubClient/AzureStorageSourceCodeClient.cs:52]: We should save the metadata to table here

Comment (rekhaswaminathan) [reply src/Diagnostics.RuntimeHost/Services/GithubClient/AzureStorageSourceCodeClient.cs:52]: https://github.com/Azure/Azure-AppServices-Diagnostics/blob/3e2c1c450407e27fa61f876ddd473e987c12d78d/src/Diagnostics.RuntimeHost/Services/SourceWatcher/Watchers/StorageWatcher.cs#L135

Comment (hforeste) [reply src/Diagnostics.RuntimeHost/Services/GithubClient/AzureStorageSourceCodeClient.cs:52]: I think your code is already updating the table

Comment (rekhaswaminathan) [reply src/Diagnostics.RuntimeHost/Services/GithubClient/AzureStorageSourceCodeClient.cs:52]: So on top of the regular updates to blob and table, you are saving each file (like metadata.json, package.json) to the blob container?

Comment (hforeste) [reply src/Diagnostics.RuntimeHost/Services/GithubClient/AzureStorageSourceCodeClient.cs:52]: Yeah specifically the detector and gist file and package.json

Comment (rekhaswaminathan) [reply src/Diagnostics.RuntimeHost/Services/GithubClient/AzureStorageSourceCodeClient.cs:52]: You can filter out dll and pdb here. Dll is saved as part of StorageWatcher and pdb is not required

**Pull request merged**